examples/vitals/tesla.proto

```diff
@@ -80,19 +363,162 @@ message Device {
     optional google.protobuf.Timestamp firstCommunicationTime = 8;
     optional google.protobuf.Timestamp lastCommunicationTime = 9;
     optional ConnectionParameters connectionParameters = 10;
-    repeated DeviceAttributes deviceAttributes = 11;
+    optional DeviceAttributes deviceAttributes = 11;
```

Comment:
Breaking change. Single value only.

Reply:
Nice! Makes sense and this cleans up the referencing.